crates/sui-graphql-rpc/src/types/move_package.rs

```diff
@@ -215,7 +221,8 @@ impl MovePackage {
         address: SuiAddress,
         version: Option<u64>,
     ) -> Result<Option<Self>, Error> {
-        let Some(object) = Object::query(db, address, version).await? else {
+        // MovePackages are immutable objects, and thus we can always pass None for checkpoint_sequence_number
```

Comment:
Note that system packages can change in place (even though they are immutable), so this is only mostly true, but it may be reasonable to ignore that detail for packages.